ci/sawtooth-build-docs

```diff
@@ -52,6 +52,7 @@ RUN echo "deb http://repo.sawtooth.me/ubuntu/nightly xenial universe" >> /etc/ap
     python3-grpcio=1.1.3-1 \
     python3-lmdb=0.92-1 \
     python3-multidict=2.1.4-1 \
+    python3-netifaces=0.10.4-0.1build2 \
```

Comment:
The version is specified in all other places this dependency is added since we are specifying a version for all dependency. So the answer to your question is no, but should be there to be consistent.